sklearn/metrics/tests/test_pairwise.py

```diff
@@ -122,6 +122,10 @@ def test_pairwise_distances():
     # Test that a value error is raised if the metric is unknown
     assert_raises(ValueError, pairwise_distances, X, Y, metric="blah")
 
+    # Test boolean metric raises dataconversion warning
+    assert_warns_message(DataConversionWarning, 'boolean', pairwise_distances,
```

Comment:
please use pytest here, and a longer match for the message

Reply:
done